modules/microshift-embed-microshift-image-offline-deploy.adoc

```diff
@@ -45,50 +45,49 @@ release-x86_64.json
 release-aarch64.json
 ----
 +
-If you installed the `microshift-release-info` RPM, you can proceed to step 4.
+If you installed the `microshift-release-info` RPM, proceed to step 4.
 
 . If you did not complete step 2, download and unpack the `microshift-release-info` RPM without installing it:
 
 .. Download the RPM package by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ sudo dnf download microshift-release-info-<release_version>
+$ sudo dnf download microshift-release-info-_<release_version>_ # <1>
 ----
-Replace `<release_version>` with the numerical value of the release you are deploying, using the entire version number, such as `4.18.1`.
+<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, such as `4.18.1`.
 +
-.Example rpm
-[source,terminal]
+.Example RPM output
+[source,terminal,subs="+quotes"]
 ----
-microshift-release-info-4.18.1.*.el9.noarch.rpm <1>
+microshift-release-info-4.18.1.-202511101230.p0.g7dc6a00.assembly.4.18.1.el9.noarch.rpm
 ----
-<1> The `*` represents the date and commit ID. Your output should contain both, for example `-202511101230.p0.g7dc6a00.assembly.4.18.1`.
 
 .. Unpack the RPM package without installing it by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ rpm2cpio <my_microshift_release_info> | cpio -idmv <1>
+$ rpm2cpio _<my_microshift_release_info>_ | cpio -idmv # <1>
 ./usr/share/microshift/release/release-aarch64.json
 ./usr/share/microshift/release/release-x86_64.json
 ----
-<1> Replace `<my_microshift_release_info>` with the name of the RPM package from the previous step.
+<1> Replace `_<my_microshift_release_info>_` with the name of the RPM package from the previous step.
 
 . Define the location of your JSON file, which contains the container reference information, by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ RELEASE_FILE=</path/to/your/release-$(uname -m).json>
+$ RELEASE_FILE=_</path/to/your/release-$(uname -m).json>_ # <1>
 ----
-Replace `</path/to/your/release-$(uname -m).json>` with the full path to your JSON file. Be sure to use the file needed for your architecture.
+<1> Replace `_</path/to/your/release-$(uname -m).json>_` with the full path to your JSON file. Be sure to use the file needed for your architecture.
 
 . Define the location of your TOML file, which contains instructions for building the image, by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ BLUEPRINT_FILE=</path/to/your/blueprint.toml>
+$ BLUEPRINT_FILE=_</path/to/your/blueprint.toml>_ # <1>
 ----
-Replace `</path/to/your/blueprint.toml>` with the full path to your JSON file.
+<1> Replace `_</path/to/your/blueprint.toml>_` with the full path to your TOML file.
```

Comment:
same

Reply:
Yes, we do not take the OCP exception, but follow the rule, https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values